tests/functions/test_reve.py

```diff
@@ -0,0 +1,43 @@
+import logging
+
+import pytest
+
+import pixeltable as pxt
+from pixeltable.functions import reve
+
+from ..utils import rerun, skip_test_if_no_client, validate_update_status
+
+_logger = logging.getLogger('pixeltable')
+
+
+@pytest.mark.remote_api
+@pytest.mark.expensive
+@rerun(reruns=3, reruns_delay=8)
+class TestReve:
```

Comment:
Worth adding some test cases that exercise the `version` parameter? Also, it's always good practice to have test cases that use default parameters only